### DISCUSSION TITLE
Groups configuration question

### DISCUSSION THREAD

**Discussion by genfish** [dachcom-digital/pimcore-members#194] [Q&A] [answered]
Hi, I'm testing the bundle in Pimcore 11, but after defining the MembersGroup class, when I try to create an object, Admin don't show me Pimcore Roles to select. Am I doing something wrong? Thanks!
![image](https://github.com/dachcom-digital/pimcore-members/assets/9285058/df376893-a055-44bd-b011-9afe3cd93e5a)
![image](https://github.com/dachcom-digital/pimcore-members/assets/9285058/4a2198f9-ac5d-4a39-9c1d-284f6a31de58)
![image](https://github.com/dachcom-digital/pimcore-members/assets/9285058/2d5e73e2-585f-4054-a271-fe676523b511)

**Comment by genfish** [comment]
I tried add more roles in the config code:

```
security:
    role_hierarchy:
        ROLE_MEMBERS_MODERATOR: [ROLE_USER]
```

And now there are 2 roles. The initial role and the role I created. But, the rest of the roles created directly in Pimcore are still missing.
![image](https://github.com/dachcom-digital/pimcore-members/assets/9285058/71bf007e-7afa-472e-b4a8-db5eaaba26ad)

**Reply by solverat** [reply]
> " the rest of the roles created directly in Pimcore"

You can't create roles in pimcore - am I missing something? 

You don't have to use roles with members, just use the group object itself to assign them to the user.

**Comment by genfish** [comment]
Then the bundle roles are completely independent of the Pimcore roles, is this so?

Let me explain, back in Pimcore 4, I developed a system that used platform users and profiles to control which documents, the user could see and access from the frontend. This, with the restriction options you've made, works fine.

But, It could also control which objects the user could see in a list, access, modify, create or delete through the folder where they were saved. That's not possible in your restriction system, no?

![image](https://github.com/dachcom-digital/pimcore-members/assets/9285058/5a33e8a2-34ca-452b-abb5-7205fd94069c)

**Reply by solverat** [reply, marked as answer]
Nope, that's not possible. Backend-Role Definitions are completely independent of frontend (which makes totally sense, I think). Every document/object/asset entity has its own Member-Restriction Tab, where you're able to configure user restrictions.

**Comment by genfish** [comment]
True, in general it makes sense, but in my case, it's about programs that administer with a different interface than the administrative one, the Pimcore objects. That's why it's important to have control of actions on objects with user profiles just as Pimcore manages it. Thanks to response, I will try it myself.
